Raven.Voron/Voron/Platform/Win32/Win32MemoryMapPager.cs

```diff
@@ -363,15 +361,36 @@ public override void MaybePrefetchMemory(List<Page> sortedPages)
 
             fixed (Win32MemoryMapNativeMethods.WIN32_MEMORY_RANGE_ENTRY* entries = list.ToArray())
             {
-                Win32MemoryMapNativeMethods.PrefetchVirtualMemory(Win32NativeMethods.GetCurrentProcess(),
+                Win32MemoryMapNativeMethods.PrefetchVirtualMemory(_currentProcess,
                     (UIntPtr)list.Count,
                     entries, 0);
             }
         }
 
+        public override void MaybePrefetchMemory(List<long> pagesToPrefetch)
+        {
+            if (CanPrefetch == false)
+                return; // not supported
+
+            if (pagesToPrefetch.Count == 0)
+                return;
+
+            var entries = new Win32MemoryMapNativeMethods.WIN32_MEMORY_RANGE_ENTRY[pagesToPrefetch.Count];
+            for (int i = 0; i < entries.Length; i++)
+            {
+                entries[i].NumberOfBytes = (IntPtr)(4 * PageSize);
```

Comment:
You want to fetch 32KB here, but it is very hard to understand.
Also, what happen if we have multiple nearby pages? Shouldn't we merge it all?